internal/datanode/flow_graph_dd_node.go

```diff
@@ -277,16 +277,17 @@ func (ddn *ddNode) Close() {
 }
 
 func newDDNode(ctx context.Context, collID UniqueID, vchanInfo *datapb.VchannelInfo,
+	flushedSegments []*datapb.SegmentInfo, unflushedSegments []*datapb.SegmentInfo, droppedSegments []*datapb.SegmentInfo,
```

Comment:
seems that we don't even need  segmentInfo here.
The only thing we can about the segment ID

Reply:
Remove flushedSegments and droppedSegments here, but still keep unflushedSegments because it is used later.